multinet/swagger/set_workspace_permissions.yaml

```diff
@@ -0,0 +1,22 @@
+Set the permissions on a workspace.
+---
+parameters:
+  - $ref: "#/parameters/workspace"
+  - name: permissions
+    description: Permission Document
+    in: body
+
+responses:
+  200:
+    description: Metadata about the specified workspace
+    schema:
+      $ref: "#/definitions/workspace"
```

Comment:
If `#/definitions/workspace` now details workspace permissions, its name should probably change to reflect that.

Reply:
98a0c81